FlowCrypt/src/main/java/com/flowcrypt/email/security/KeysStorageImpl.kt

```diff
@@ -237,6 +239,17 @@ class KeysStorageImpl private constructor(context: Context) : KeysStorage {
     return passPhraseMap.filter { it.value.passphrase.isEmpty }.map { it.key }
   }
 
+  override fun getFirstUsableForEncryptionPGPSecretKeyRing(user: String): PGPSecretKeyRing? {
+    val keyRings = getPGPSecretKeyRingsByUserId(user)
```

Comment:
probably could be shorter
`return getPGPSecretKeyRingsByUserId(user).firstOrNull { KeyRingInfo(it).usableForEncryption()  }`